docs/platform/concepts/cloud-security.rst

```diff
@@ -77,4 +77,14 @@ Aiven operations team has mandatory recurring training regarding the applicable
 Periodic security evaluation
 -----------------------------
 
-Aiven services are periodically assessed, and penetration tested for any security issues by an independent professional cyber security vendor.
\ No newline at end of file
+Aiven services are periodically assessed, and penetration tested for any security issues by an independent professional cyber security vendor.
+
+
+SBOM(Software Bill of Materials)
+--------------------------------
+
+A SBOM is a list of all packages that are being used by Aiven in the services we provide. The list comprehend packages installed in the Node and VM operating system, also the packages installed by the service itself.
+
+SBOMs is being increasingly adopted and might be required for compliance reasons. We provide the report to be download through our CLI as CSV of SPDX files. :doc:`here <docs/tools/cli/project.rst>`
+
+Not all customers are allowed to generate/download SBOMs, if you need an SBOM report from us please reach out to <WHERE?>
```

Comment:
I'll mark the PR as draft until all the information is available, please update when you're ready.